blobs/howto/dotnet/dotnet-v12/Authorize_DataLake.cs

```diff
@@ -34,15 +34,12 @@ public static class Authorize_DataLake
 
         // <Snippet_AuthorizeWithAAD>
```

Comment:
Since we're making changes, I might move away from the ref parameter usage. Elsewhere we create the client in the function and return it. Something like:

```
DataLakeServiceClient GetDataLakeServiceClient(String accountName)
{
    string dfsUri = $"https://{accountName}.dfs.core.windows.net";

    DataLakeServiceClient dataLakeServiceClient = new DataLakeServiceClient(
        new Uri(dfsUri),
        new DefaultAzureCredential());

    return dataLakeServiceClient;
}
```

Reply:
Done